external/webrtc-sdk.js

```diff
@@ -32691,9 +32691,17 @@ define("WebRTC_SDK/manager/RTCLocalMediaManager", [
       var deferred = Q.defer();
 
       var stream = this.model.get("localAudioStream");
+      if (_.isNull(stream) || _.isUndefined(stream)) {
+        Logger.warn(
+          "Local Audio stream is null, only mute/unmute local backbone model."
+        );
+        this.model.set("localAudioMuted", mute);
```

Comment:
This is the important part, if the stream does not exist, we cannot mute the streams. However, we can still mute Backbone's internal model that keeps track of the mute state.

Reply:
This was not being set before.